### PR TITLE
truffle integration

### PR DESCRIPTION
**Pull request by Aniket-Engg**

### CODE REVIEW

File: libs/remix-ui/solidity-compiler/src/lib/logic/compileTabLogic.ts [hunk 1]
```diff
@@ -133,6 +139,12 @@ export class CompileTabLogic {
             this.api.logToTerminal({ type: 'error', value: error })
           })
         }
+      } else if (externalCompType === 'truffle') {
```
Comment (yann300) [inline libs/remix-ui/solidity-compiler/src/lib/logic/compileTabLogic.ts:142]: `this.api.getFileManagerMode() === 'localhost'` should be in the check too.

**Comment by yann300** [comment]
could you rollback the changes to package-lock.json

File: libs/remix-ui/solidity-compiler/src/lib/logic/compileTabLogic.ts (hunk 1 above)
Comment (Aniket-Engg) [reply libs/remix-ui/solidity-compiler/src/lib/logic/compileTabLogic.ts:142]: ah yes

Comment (Aniket-Engg) [reply libs/remix-ui/solidity-compiler/src/lib/logic/compileTabLogic.ts]: updated

**Comment by Aniket-Engg** [comment]
> could you rollback the changes to package-lock.json

done

**Review by yann300 — APPROVED** [review]

**Pull request merged**